src/main/java/org/elasticsearch/cluster/InternalClusterInfoService.java

```diff
@@ -316,12 +332,18 @@ public void onResponse(NodesStatsResponse nodeStatses) {
 
                 @Override
                 public void onFailure(Throwable e) {
-                    if (e instanceof ClusterBlockException) {
-                        if (logger.isTraceEnabled()) {
-                            logger.trace("Failed to execute NodeStatsAction for ClusterInfoUpdateJob", e);
-                        }
+                    if (e instanceof ReceiveTimeoutTransportException) {
+                        logger.error("NodeStatsAction timed out for ClusterInfoUpdateJob (reason [{}])", e.getMessage());
                     } else {
-                        logger.warn("Failed to execute NodeStatsAction for ClusterInfoUpdateJob", e);
+                        if (e instanceof ClusterBlockException) {
+                            if (logger.isTraceEnabled()) {
```

Comment:
Yeah, I was saying we can remove it now, since I was the one that originally added it